Core/Core/Common/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -64,3 +64,11 @@ public extension Array where Element: Encodable {
         try? JSONEncoder().encode(self)
     }
 }
+
+public extension Array {
+    func chunked(into size: Int) -> [[Element]] {
```

Comment:
```suggestion
extension Array {
    public func chunked(into size: Int) -> [[Element]] {
```

Just doublechecking: Is this covered indirectly with unit tests?